extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -58,6 +58,22 @@ const query_databases_with_size = `
 	WHERE sys.databases.state = 0
 	`;
 
+const query_login_tables_sql = `
+	SELECT
+		sp.name as login,
+		sp.type_desc as login_type,
+		sp.default_database_name,
+		case when sp.is_disabled = 1 then 'Disabled' else 'Enabled' end as status
+	FROM sys.server_principals sp
+	LEFT JOIN sys.sql_logins sl ON sp.principal_id = sl.principal_id
+	WHERE sp.type NOT IN ('G', 'R') AND sp.type_desc IN (
+		'SQL_LOGIN'
+		--, 'WINDOWS_LOGIN'
```

Comment:
can this be removed?

Reply:
So this will be needed back in once we figure out windows auth. I was trying to add a comment, but couldn't find an easy, clean way to do it here